corehq/apps/reports/templates/reports/form/partials/single_form_tree.html

```diff
@@ -7,6 +7,12 @@
                     <strong>{% trans "Hidden Values (Form Calculations)" %}</strong>
                 </td>
             </tr>
+        {% elif 'commcare_usercase/' in question.label %}
```

Comment:
It could be. They contain the same string. Would `question.value` be more reliable?